keyboards/treadstone48/keymaps/default/keymap.c

```diff
@@ -14,16 +14,13 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include QMK_KEYBOARD_H
-#include "keymap_jp.h"
 #include "../common/oled_helper.h"
 
 #ifdef RGBLIGHT_ENABLE
 //Following line allows macro to read current RGB settings
 extern rgblight_config_t rgblight_config;
 #endif
```

Comment:
This also shouldn't be needed.

It's for grabbing the rgb light mode, but you can use `rgblight_get_mode()` instead.

Reply:
thank you.